pkg/subctl/cmd/logs.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+)
+
+func getGatewayPodsDetails(clientSet kubernetes.Interface) {
```

Comment:
The same logic will be replicated for lighthouse, routeagent and operator pods once this is finalised.

Reply:
This will be invoked from subctl gather command that @vthapar is working on PR #1159